docs/docs/concepts/redirects.mdx

```diff
@@ -1,9 +1,146 @@
+only allowing certain domains, or paths on a domain. As shown in this example
+below the https://sub.domain.myapp.com/only/path needs to match perfectly for
+this sub-domain and path for it to be allowed. Other redirects using myapp.com
+will fail, however, for https://anotherapp.com will succeed on any path.
+
+![Browser Redirects Allow List](../images/browser-redirects/browser-redirects-allow-list.png)
+
+## Redirect Flows
+
+The Ory Cloud has a total of 6 flows, Login, Registration, Verification,
+Recovery, Settings and Logout which can be configured to redirect back to any
+url you specify. As an example you can redirect the user to your application
+home screen after a logout or to a specific url on a sub-domain after a settings
+password update. On project creation the redirect flows are all set to the
+default Ory Cloud Managed UI. For more information on the Ory Managed UI, please
+see the [Managed User Interface](./managed-ui.mdx) documentation.
+
+### Login, Registration and Settings
+
+The Login and Registration flows have three fields (all optional):
+
+1. `Default Redirect Url` domain or path
+2. An `Post-Password Redirect Url`
+3. An `Post-OIDC Redirect Url` (**currently not supported**)
+
+And Settings has three fields (all optional):
+
+1. `Default Redirect Url` domain or path
+2. An `Post-Password Redirect Url`
+3. An `Post-Profile Redirect Url`
+
+:::info
+
+Setting `Post-OIDC` redirects for Login and Registration are currently not
+supported since an Ory Cloud project cannot be setup to use OIDC yet. This will
+be supported in the future. For more information see the
+[Ory Cloud Roadmap](../early-access.mdx).
+
+:::
+
+When setting the `Default Redirect Url` as a FQDN (fully qualified domain name)
+such as e.g. https://domain.example/login it will overwrite the
+`Global Redirect Url`. As a path it will replace the `Global Redirect Url` path,
+e.g. a path such as `/login` will alter https://domain.example/global/path to
+https://domain.example/login.
+
+On the sub-conditions such as `Post-Password` or `Post-OIDC` a FQDN or a path
+can be set. In the case of a FQDN it will replace the closest relative starting
+with the Login/Registration `Default Redirect Url` and then the
+`Global Redirect Url`. With a path it will use the closest relative as the base
+url as described above with the `Default Redirect Url`.
+
+![Browser Redirects Url Management](../images/browser-redirects/browser-redirects-url-management.png)
+
+All the relative urls will be updated with their respective base url. As shown
+in the example below, the Login `Default Redirect Url` has been set to use a
+relative url and thus the base url will be the `Global Redirect Url`
+(https://myapp.com). This will then set the Login `Default Redirect Url` to
+https://myapp.com/after/login and the `Post-Password` to
+https://myapp.com/after/login/password.
+
+![Browser Redirects Url Management Transformed](../images/browser-redirects/browser-redirects-url-management-2.png)
```

Comment:
```suggestion
All the relative URLs will be updated with their respective base URL. As shown
in the example below, the Login `Default Redirect URL` has been set to use a
relative URL and thus the base URL will be the `Global Redirect URL`
(https://myapp.com). This will then set the Login `Default Redirect URL` to
https://myapp.com/after/login and the `Post-Password` to
https://myapp.com/after/login/password.

![Browser Redirects URL Management Transformed](../images/browser-redirects/browser-redirects-url-management-2.png)
```